tests/travis.sh

```diff
@@ -45,7 +48,7 @@ if [ "$result" != "" ]; then
 	echo "$result"
 	echo Some tests did not complete successfully, changing build status to failed
 	exit 1
-elif [ $(grep -c "ERROR\|CRITICAL\|EMERGENCY" "$DATA_DIR/server.log") -ne 0 ]; then
+elif [ "$(grep -c "ERROR\|CRITICAL\|EMERGENCY" "$DATA_DIR/server.log")" -ne 0 ]; then
```

Comment:
Do we not need to escape the inner quotes here?

Reply:
Maybe the CI failling if not ? I wouldn't know why else

Reply:
Yeah you're right. The PR is its own proof of validity I guess.

Reply:
Looks like there's no escaping needed according to this: https://unix.stackexchange.com/questions/118433/quoting-within-command-substitution-in-bash